### ISSUE TITLE
File not found exception because converter can't find textures files even though they are there

### ISSUE THREAD

**Issue by Sandwichas** [Simsilica/JmeConvert#2]
```
[Convert] Max memory:3641.00 mb
15:09:32,566 INFO  [AssetReader] Using source asset root:D:\Software\Development and Programming\JmeConvert\Models\Original
15:09:32,567 INFO  [AssetReader] Found assetConfig:jar:file:/D:/Software/Development%20and%20Programming/JmeConvert/jmec-1.3.0/lib/jme3-core-3.3.2-stable.jar!/com/jme3/asset/Desktop.cfg
15:09:32,620 WARN  [AssetConfig] Cannot find loader com.jme3.scene.plugins.blender.BlenderModelLoader
15:09:32,632 WARN  [AssetConfig] Cannot find loader com.jme3.audio.plugins.OGGLoader
15:09:32,634 INFO  [Convert] Convert:D:\Software\Development and Programming\JmeConvert\Models\Original\indoor plant_02_+2.fbx
15:09:32,635 INFO  [AssetReader] Loading asset:indoor plant_02_+2.fbx
15:09:32,664 WARN  [FbxGlobalSettings] FBX model isn't using correct forward axis sign. Orientation could be incorrect
15:09:32,665 WARN  [FbxGlobalSettings] FBX model isn't using meters for world units. Scale could be incorrect.
15:09:32,680 WARN  [FbxLoader] An object with ID "450424641" has already been defined. Ignoring.
15:09:32,681 WARN  [FbxLoader] An object with ID "473526158" has already been defined. Ignoring.
15:09:32,682 WARN  [FbxLoader] An object with ID "450424641" has already been defined. Ignoring.
15:09:32,682 WARN  [FbxLoader] An object with ID "473526158" has already been defined. Ignoring.
15:09:32,683 WARN  [FbxLoader] An object with ID "450424641" has already been defined. Ignoring.
15:09:32,684 WARN  [FbxLoader] An object with ID "473526158" has already been defined. Ignoring.
15:09:32,981 INFO  [Convert] Processing:indoor plant_02_+2.fbx
15:09:32,982 INFO  [AssetWriter] Copying:D:\Software\Development and Programming\JmeConvert\Models\Original\indoor plant_2_NOR.jpg (Flipped) (Mipmapped) to:D:\Software\Development and Programming\JmeConvert\Models\Converted\assets\indoor plant_2_NOR.jpg (Flipped) (Mipmapped)
Exception in thread "main" java.lang.RuntimeException: Error writing model:indoor plant_02_+2.fbx
        at com.simsilica.jmec.AssetWriter.apply(AssetWriter.java:97)
        at com.simsilica.jmec.Convert.runProcessors(Convert.java:286)
        at com.simsilica.jmec.Convert.convert(Convert.java:274)
        at com.simsilica.jmec.Convert.main(Convert.java:335)
Caused by: java.io.FileNotFoundException: D:\Software\Development and Programming\JmeConvert\Models\Original\indoor plant_2_NOR.jpg (Flipped) (Mipmapped) (The system cannot find the file specified)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at com.google.common.io.Files$FileByteSource.openStream(Files.java:126)
        at com.google.common.io.Files$FileByteSource.openStream(Files.java:116)
        at com.google.common.io.ByteSource.copyTo(ByteSource.java:267)
        at com.google.common.io.Files.copy(Files.java:315)
        at com.simsilica.jmec.AssetWriter.write(AssetWriter.java:117)
        at com.simsilica.jmec.AssetWriter.apply(AssetWriter.java:95)
        ... 3 more
```
When trying to convert any 3D object with textures, the operation fails with this exception. At first, I thought that my .obj file was the problem. However, it still doesn't work even with other objects I downloaded online to test.

The files can be seen in the following picture:
![Screenshot_7](https://user-images.githubusercontent.com/14120627/171998499-334ee186-cb27-4208-97b0-76cfa51d74d0.png)
The converted files path (empty):
![Screenshot_8](https://user-images.githubusercontent.com/14120627/171998537-8802881f-eed9-4d50-bb46-bd888b000e35.png)
Command used:
`jmec.bat -sourceRoot "D:\Software\Development and Programming\JmeConvert\Models\Original" -targetRoot "D:\Software\Development and Programming\JmeConvert\Models\Converted" -targetPath "assets" "D:\Software\Development and Programming\JmeConvert\Models\Original\indoor plant_02_+2.fbx"`

**Comment by pspeed42** [comment]
Pretty sure this is related to a bug I fixed recently that has not made it into a release.  The issue is that the file: "D:\Software\Development and Programming\JmeConvert\Models\Original\indoor plant_2_NOR.jpg (Flipped) (Mipmapped)"
...really doesn't exist because the real file (obviously) doesn't have "(Flipped)" and "(Mipmapped)" on the end.  It's a bug in JMEC with how it extracts the asset file name from the asset key.

It was fixed in this commit: https://github.com/Simsilica/JmeConvert/commit/54dda843d70a7c922075062e9aa8f780c7f8ba94

Sorry for the trouble.  If it's super important then I can try cutting a new release this weekend but it's also pretty easy to build from source if you clone the repo:
gradlew distZip
